hack/go-build.sh

```diff
@@ -80,7 +81,7 @@ if [[ -z ${CI} ]]; then
     ./hack/go-test.sh
     BUILD_PARAMS=""
     if [[ ! -z ${BUILDER} ]]; then BUILD_PARAMS="${BUILD_PARAMS} --image-builder ${BUILDER}"; fi
-    operator-sdk build ${REGISTRY}/${IMAGE}:${TAG} ${BUILD_PARAMS}
+    operator-sdk build ${REGISTRY}/${IMAGE}:${TAG} ${BUILD_PARAMS} --go-build-args -o=${BINARY_OUTPUT}
```

Comment:
this will force the output name to `kogito-cloud-operator` because default, operator-sdk is taking the folder name for the binary and it may not run in a `kogito-cloud-operator` named folder